### PR TITLE
REV-90 add show password into signup and login

### PR DESCRIPTION
**Pull request by IbrahimAljabrST**
### Jira URL

https://optimumpartners.atlassian.net/browse/REV-90

### Description

add eye for login and sign up for password

### CODE REVIEW

File: src/pages/Login/login.styles.js [hunk 1]
```diff
@@ -66,6 +66,36 @@ export const InputWrapper = styled.div`
...
+    width: 100%;
+  }
+  input {
+  }
+  svg {
+    bottom: 0.75rem;
+    height: 0.9rem;
+    position: absolute;
+    right: 0.8rem;
+    width: 1.6rem;
```
Comment (AhmadAlShwaiki) [inline src/pages/Login/login.styles.js:84]: Please use styled components.

File: src/pages/Login/Login.jsx [hunk 2]
```diff
@@ -35,6 +39,7 @@ import {
  */
 function Login() {
   const [error, setError] = useState(false);
+  const [passwordOn, setPasswordOn] = useState(false);
```
Comment (mohdabed123) [inline src/pages/Login/Login.jsx:42]:
```suggestion
  const [isShowPassword, setIsShowPassword] = useState(false);
```

File: src/pages/Login/login.styles.js [hunk 3]
```diff
@@ -66,6 +66,37 @@ export const InputWrapper = styled.div`
   margin-bottom: 1.25rem;
 `;
 
+export const InputGroup = styled.div`
+  display: flex;
+  align-items: center;
+  position: relative;
+`;
+
+export const PasswordInputRapper = styled.div`
```
Comment (AhmadAlShwaiki) [inline src/pages/Login/login.styles.js:75]: Cant see why this is needed

File: src/pages/Login/login.styles.js [hunk 4]
```diff
@@ -66,6 +66,37 @@ export const InputWrapper = styled.div`
...
+  width: 100%;
+`;
+
+export const EyeIconRapper = styled.div`
```
Comment (AhmadAlShwaiki) [inline src/pages/Login/login.styles.js:79]: pretty sure I saw this around the app
```suggestion
export const IconContainer = styled.div`
```

File: src/pages/Login/login.styles.js (hunk 3 above)
Comment (IbrahimAljabrST) [reply src/pages/Login/login.styles.js:75]: its container for input and icon only for the shared component with label

**Review by AhmadAlShwaiki — APPROVED** [review]

**Review by mohdabed123 — APPROVED** [review]

File: src/pages/Login/login.styles.js (hunk 3 above)
Comment (AhmadAlShwaiki) [reply src/pages/Login/login.styles.js:75]: Please add styles to inputGroup and remove this component

File: src/pages/Login/Login.jsx [hunk 5]
```diff
@@ -82,15 +86,24 @@ function Login() {
...
-              name="password"
-              error={errors.password?.message}
-              label="Password"
-              register={register}
-              type="password"
-              onChange={() => setFocus('password')}
-            />
-            {error && <ErrorMessage>Invalid email or password</ErrorMessage>}
+            <InputGroup>
+              <div>
```
Comment (AhmadAlShwaiki) [inline src/pages/Login/Login.jsx:90]:
```suggestion
```

File: src/pages/Login/login.styles.js [hunk 6]
```diff
@@ -66,6 +66,35 @@ export const InputWrapper = styled.div`
...
 `;
 
+export const InputGroup = styled.div`
+  display: flex;
+  align-items: center;
+  position: relative;
+
+  div:first-child {
+    width: 100%;
+  }
```
Comment (AhmadAlShwaiki) [inline src/pages/Login/login.styles.js:76]:
```suggestion
```

File: src/pages/Login/Login.jsx (hunk 5 above)
Comment (AhmadAlShwaiki) [inline src/pages/Login/Login.jsx:90]:
```suggestion
```
Can't see why this is needed

**Pull request merged**